packages/atlas/src/views/global/YppLandingView/YppThreeStepsSection.styles.ts

```diff
@@ -76,3 +83,14 @@ export const StepCardFade = styled.div`
   width: 100%;
   bottom: 0;
 `
+
+export const StyledBackgroundContainer = styled(BackgroundContainer)`
+  padding: 0;
+`
+
+export const StyledLimitedWidthContainer = styled(_StyledLimitedWidthContainer)`
```

Comment:
This could result in importing the wrong component by accident in this case. I would name this differently.

Reply:
These styles are actually unused, I'll remove them